tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -0,0 +1,111 @@
+*** Settings ***
+Library         OpenShiftCLI
+Resource        ../../Resources/ODS.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboardResources.resource
+Resource        ../../Resources/Page/LoginPage.robot
+Test Setup      Resources Test Setup
+Test Teardown   Resources Test Teardown
+
+
+*** Test Cases ***
+Verify Quick Starts Work As Expected
```

Comment:
Maybe better just `Verify Dashboard Quick Starts`. "Work As Expected" is not very meaningful

Reply:
Actually, I'm thinking about of splitting this TC in more TCs, in that case we will not have the problem about the name anymore